src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -505,7 +505,8 @@ private static void SetSslCertificate(SafeSslContextHandle contextPtr, SafeX509H
 
         internal static SslException CreateSslException(string message)
         {
-            ulong errorVal = Crypto.ErrGetError();
+            ulong errorVal = Crypto.ErrPeekLastError();
```

Comment:
Why change Get to Peek?

Reply:
Better question, why change first to last?  the first error should be the most relevant.  It's the thing that went wrong.

The last error is probably caused by the first one.

Reply:
I guess a good reason would be because that's what CreateOpenSslCryptographicException does.

Reply:
Yes, but to be fair this will change what it was doing in the past... but I think while it risks not showing the root cause at least it has more chance of seeing related to the call at hand. However, now that we are keeping the error queue clean, we could, in principle, show every error - but that won't be for now.